docker/Dockerfile.agent.multiarch.rootless

```diff
@@ -1,4 +1,8 @@
-FROM --platform=$BUILDPLATFORM docker.io/golang:1.23 AS build
+FROM --platform=$BUILDPLATFORM docker.io/golang:1.23 AS builder
+
+RUN adduser woodpecker && \
+  mkdir -p /var/lib/woodpecker && \
```

Comment:
Why is this directory created here? Looks like its never used and also doesnt exist in the alpine image.